### ISSUE TITLE
[SampleCard-FlightDetails.json; Config-Default]: Color Contrast ratio for 'Departing Airplane Image' is 1.57:1 and should be at least 3:1.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7372]
https://microsoft.visualstudio.com/OS/_workitems/edit/37984923
